docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/amplitude-tag-for-gtm-ss/amplitude-tag-configuration/index.md

```diff
@@ -70,9 +68,15 @@ Depending on the value set for the **Apply to all versions** column, the major v
 
 :::
 
-:::info
+<details>
+
+<summary><i>pre-v0.2.0</i></summary>
```

Comment:
nothing wrong with this, but just FYI I think you can still use Markdown instead of `<i>`